src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -294,6 +294,7 @@ internal enum MessageID
         IDS_OverloadResolutionPriority = MessageBase + 12848,
 
         IDS_FeatureFirstClassSpan = MessageBase + 12849,
+        IDS_FeatureUnboundGenericTypesInNameof = MessageBase + 12850,
```

Comment:
Some paperwork: You'll want to add an entry at the top of "Working Set C#" section on https://github.com/dotnet/roslyn/blob/main/docs/Language%20Feature%20Status.md

Reply:
More paperwork :-P You'll want to convert the championed proposal to new format (ie. move the spec portion into a speclet document in `proposals` folder)

Reply:
will do :)